src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -78,9 +78,14 @@
                               CommandParameter="CrossProduct"
                               Header="{x:Static p:Resources.ContextMenuLacingCrossProduct}" />
                 </MenuItem>
+                <MenuItem Name="nodeIsFrozen"
+                          Header="{x:Static p:Resources.NodesRunStatus}"
+                          IsChecked="{Binding IsExplicitFrozen,Mode=OneWay}"   
+                          IsEnabled="{Binding CanToggleFrozen, Mode=OneWay}"
+                          Command="{Binding Path=ComputeRunStateOfTheNodeCommand}" />
```

Comment:
If we remove this command, then how IsFrozen on NodeModel will be toggled? this command is used only on the nodeview, that toggles IsFrozen. I believe this command should live on NodeView.